File: android/trunk/src/com/usedparts/domain/Offer.java
package com.usedparts.domain;
import com.fasterxml.jackson.annotation.JsonFormat;
import com.fasterxml.jackson.annotation.JsonProperty;

import java.util.*;

public class Offer
{
    @JsonProperty("ix")
    public long id;

    @JsonProperty("cond")
    public int condition;

    @JsonProperty("deli")
    public int delivery;

    @JsonProperty("gara")
    public int warranty;

    @JsonProperty("ava")
    public int available;

    @JsonProperty("price")
    public double price;

    @JsonProperty("offDate")
    @JsonFormat(shape=JsonFormat.Shape.STRING, pattern="dd.MM.yyyy")
    public Date posted;

    @JsonProperty("sellerHead")
    public String selllerName;

    @JsonProperty("sellerLogo")
    public String selllerLogo;

    @JsonProperty("usrId")
    public int sellerId;

    @JsonProperty("reqId")
    public long orderId;

    @JsonProperty("imgs")
    public List<String> images;

    @JsonProperty("thumbs")
    public List<String> thumbs;
}

File: wp/trunk/UsedParts.Domain/Offer.cs
using System;
using System.Collections.Generic;
using Newtonsoft.Json;
using UsedParts.Domain.Converters;

namespace UsedParts.Domain
{

    public class Offer
    {
        [JsonProperty(PropertyName = "ix")]
        public long Id { get; set; }

        [JsonProperty(PropertyName = "cond")]
        public int Condition { get; set; }

        [JsonProperty(PropertyName = "deli")]
        public int Delivery { get; set; }

        [JsonProperty(PropertyName = "gara")]
        public int Warranty { get; set; }

        [JsonProperty(PropertyName = "ava")]
        public int Available { get; set; }

        [JsonProperty(PropertyName = "price")]
        public double Price { get; set; }

        [JsonProperty(PropertyName = "offDate")]
        [JsonConverter(typeof(DateConverter))]
        public DateTime Posted { get; set; }

        [JsonProperty(PropertyName = "sellerHead")]
        public string SelllerName { get; set; }

        [JsonProperty(PropertyName = "sellerLogo")]
        public string SelllerLogo { get; set; }

        [JsonProperty(PropertyName = "usrId")]
        public int SellerId { get; set; }

        [JsonProperty(PropertyName = "reqId")]
        public int OrderId { get; set; }

        [JsonProperty(PropertyName = "imgs")]
        public IEnumerable<string> Images { get; set; }

    }

}
